cognite/neat/rules/models/_rules/dms_schema.py

```diff
@@ -67,6 +85,171 @@ def from_model_id(cls, client: CogniteClient, data_model_id: dm.DataModelIdentif
             containers=containers.as_write(),
         )
 
+    @classmethod
```

Comment:
is this modular data model loading? like we discussed?

Reply:
Not sure what you are hinting to here.